http-api-monitor-offers.adoc

```diff
@@ -0,0 +1,327 @@
+
+    docker run -it --rm -p 8080:8080 -e BISQ_API_HOST=0.0.0.0 bisq/api
+
+=== Run the API from source
+
+For more hard-core developers that want to run from source:
+
+    git clone https://github.com/mrosseel/bisq-api
+    cd bisq-api
+    mvn compile exec:java \
+        -Dexec.mainClass="io.bisq.api.app.ApiMain" \
+        -Dexec.args="--appName=bisqmon"
+
+=== Verify the API is running
+
+In both cases the API should be running on port 8080. You can verify that by executing:
+
+    curl http://localhost:8080/api/v1/version
+
+You should receive a response like the following:
+
+[source,json]
+----
+{
+    "application": "0.6.7",
+    "network": 1,
+    "p2PMessage": 10,
+    "localDB": 1,
+    "tradeProtocol": 1
+}
+----
+
+[NOTE]
+Complete interactive API documentation is available at http://localhost:8080/swagger.
+
+== List available offers
+
+Open offers are available at http://localhost:8080/api/v1/offers.
+
+The response should look like this:
+
+[source,json]
+----
+{
+    "offers": [OfferDetail],
+    "total": integer
+}
+----
+
+Where `offers` is an array with individual offers and `total` is number of all offers. The model for each individual `OfferDetail` is defined as follows:
+
+[source,json]
+----
+{
+    acceptedBankIds	[string]
+    acceptedCountryCodes	[string]
+    amount	integer($int64)
+    arbitratorNodeAddresses	[string]
+    bankId	string
+    baseCurrencyCode	string
+    blockHeightAtOfferCreation	integer($int64)
+    buyerSecurityDeposit	integer($int64)
+    counterCurrencyCode	string
+    countryCode	string
+    currencyCode	string
+    date	string($date-time)
+    direction	string Enum: [ BUY, SELL ]
+    hashOfChallenge	string
+    id	string
+    isCurrencyForMakerFeeBtc	boolean
+    isPrivateOffer	boolean
+    lowerClosePrice	integer($int64)
+    makerFee	integer($int64)
+    makerPaymentAccountId	string
+    marketPriceMargin	number($double)
+    maxTradeLimit	integer($int64)
+    maxTradePeriod	integer($int64)
+    minAmount	integer($int64)
+    offerFeePaymentTxId	string
+    ownerNodeAddress	string
+    paymentMethodId	string
+    price	integer($int64)
+    protocolVersion	integer($int32)
+    sellerSecurityDeposit	integer($int64)
+    state	string Enum: [ UNKNOWN, OFFER_FEE_PAID, AVAILABLE, NOT_AVAILABLE, REMOVED, MAKER_OFFLINE ]
+    txFee	integer($int64)
+    upperClosePrice	integer($int64)
+    useAutoClose	boolean
+    useMarketBasedPrice	boolean
+    useReOpenAfterAutoClose	boolean
+    versionNr	string
+}
+----
+
+Now let's assume that we want to buy bitcoin, and let's define "interesting" offers as those that:
+
+ . sell bitcoin at a discount 1% or more under the current market price, and
+ . accept payment in Euros to a Polish SEPA account
+
+First we need to filter those offers using following static criteria:
+
+[source,json]
+----
+{
+    baseCurrencyCode: 'BTC',
+    counterCurrencyCode: 'EUR',
+    direction: 'SELL',
+    paymentMethodId: 'SEPA'
+}
+----
```

Comment:
This result and the "we need to filter those offers" language above again make me as a reader feel like I'm supposed to _do_ something with this information. There is actually no "step" here, it's just for informational / context purposes, and that's fine, but it should be made more clear. For example, you can say something like:

> We'll need to filter our offers using the static criteria you see below:
>
> [json snippet]
>
> In the sections that follow, we'll do just that with a combination of data returned from Bisq HTTP API endpoint calls and programmatic filtering.

(Very quickly written, but you get the idea)

Reply:
`List available offers` and `Get the market price` parts are just describing how the API looks like.
I wanted to first give API overview and then dive into the javascript.

Reply:
I've introduced `API overview` section and those `List available offers` and `Get the market price` are now subsections so it should be clear for the user that they don't need to do anything there.